tests/testthat/test-geom-sf.R

```diff
@@ -0,0 +1,14 @@
+context("geom-sf")
+
+# Visual tests ------------------------------------------------------------
+
+test_that("geom_sf draws correctly", {
+  library(sf)
```

Comment:
Since sf is only suggests, this should start with `skip_if_not_installed("sf")`

I think it's slightly nicer to not attach the package, but just `sf::read_sf()`